### ISSUE TITLE
Accessibility: Insufficient color contrast (multiple elements)

### ISSUE THREAD

**Issue by nmpegetis** [nmpegetis/nbegetis.com-gatsby#18]
Pa11y found several insufficient contrast failures (WCAG 1.4.3) across the site, notably nav links and small inline colored spans. Example selectors: '#responsive-navbar-nav > div > a:nth-child(1)', '#gatsby-focus-wrapper > div > div:nth-child(2) > div > div:nth-child(1) > p:nth-child(10) > i > strong:nth-child(1) > span:nth-child(1)'.\nSuggested fixes:\n- Update color tokens/variables to meet a minimum 4.5:1 contrast ratio for body text and navigation links.\n- Audit other color variants (badges, inline spans) and update CSS / theming as needed.\nPa11y artifact: https://github.com/nmpegetis/nbegetis.com-gatsby/actions/runs/20480073105/artifacts/4959501879
